src/RelayClient.ts

```diff
@@ -313,6 +324,8 @@ export class RelayClient {
                     relayWorkerAddress,
                     feesReceiver
                 )) + Number(trxDetails.tokenGas);
+
+            //FIXME check why two times its being multiply by ESTIMATED_GAS_CORRECTION_FACTOR
```

Comment:
Is this still valid? do we plan to fix it in a different task?

Reply:
fixed